.cirrus.yml

```diff
@@ -24,11 +21,19 @@ environment:
   CIRRUS_CHANGE_MESSAGE: ""
   CIRRUS_COMMIT_MESSAGE: ""
 
+
+gcp_credentials: ENCRYPTED[1917385f09764ac58515f6fee14ed58688afc808849b244969d41d0b082061eecd0a4c8ba801f00833877a12b51f8bb5]
+
+
 # LINUX SHARDS
 task:
-  container:
-    # https://cirrus-ci.org/guide/docker-builder-vm/#dockerfile-as-a-ci-environment
+  gke_container:
     dockerfile: "dev/ci/docker_linux/Dockerfile"
+    builder_image_name: docker-builder # gce vm image
```

Comment:
I'm guessing this is the same VM that we use to build docker images for the engine?

Reply:
Yes it is the same. The idea is to make all the repos use Cirrus in a similar way and encapsulate them to a single service account and gcp project.

Reply:
SGTM